torch/optim/rmsprop.py

```diff
@@ -3,14 +3,20 @@
 
 
 class RMSprop(Optimizer):
-    """Implements RMSprop algorithm.
+    r"""Implements RMSprop algorithm.
 
     Proposed by G. Hinton in his
     `course <http://www.cs.toronto.edu/~tijmen/csc321/slides/lecture_slides_lec6.pdf>`_.
 
     The centered version first appears in `Generating Sequences
     With Recurrent Neural Networks <https://arxiv.org/pdf/1308.0850v5.pdf>`_.
 
+    The implementation here takes the square root of the gradient average before
+    adding epsilon (note that Tensor Flow interchange these two operations). The effective
```

Comment:
TensorFlow is one word, I think. Also, I think this should be "note that TensorFlow interchanges these two operations"

Reply:
Oops! Thanks :)